js/src/interface.js

```diff
@@ -225,7 +226,8 @@ let app = new Vue({
     selectedDataSource() {
       var dataSource = this.selectedDataSource;
       this.onSelectChange();
-      if (dataSource) {
+      if (dataSource && !firstLoad) {
+        firstLoad = false;
```

Comment:
@tonytlwu 
If theres a data source and it is not the first load CLEAN columns.
If there's a data source and is is the first load DO NOT CLEAN columns. (we want to load them as they were the previouly saved ones by the user)

Actually you did add the columns clean up, and on first load the previously selected columns wasn't showing up.